docs/README.md

```diff
@@ -38,181 +38,92 @@ Code | Text | Description
 
 ### Headers and request signing
 
-All API calls need to be signed using HMAC and SHA256 or SHA512.
+All API calls need to be signed using HMAC and SHA-256 or SHA-512. When a request contains a body, the body must be valid JSON and `content-type` header with value `application/json; charset=utf-8` should be included. Other encodings may work too, but using UTF-8 is preferred.
 
-HMAC is calculated from the HTTP request's body payload using the signing secret. Header keys must be sorted alphabetically, lower-cased and included in HMAC calculation.
+All API responses are signed the same way allowing merchant to verify response validity. In addition the reponses contain `cof-request-id` header. Saving or logging the value of this header is recommended.
 
-* The merchant ID needs to be in `Checkout-Account`
-* The algorithm needs to be in `Checkout-Algorithm` (SHA256 or SHA512)
-* The calculated HMAC needs to be sent in each request in the HTTP header `Checkout-Signature`
-* The method header `Checkout-Method` needs to be set to `POST`, `GET` or `DELETE`
+The signature is transmitted in `signature` HTTP header. Signature payload consists the following fields separated with a line feed (\n). Carrige returns (\r) should not be used.
```

Comment:
We should remove conditionals from the documentation, state them as truths. This is the single source of truth for the integrating party.

Carriage returns (\r) are not supported.